bundle/manifests/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -151,7 +244,7 @@ metadata:
       Vault.
     operatorframework.io/suggested-namespace: vault-config-operator
     operators.openshift.io/infrastructure-features: '["Disconnected"]'
-    operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.13.1
```

Comment:
can you add this: `operatorframework.io/cluster-monitoring: "true"` here? It should have been there.

Reply:
Removed.